src/components/Radios.tsx

```diff
@@ -0,0 +1,290 @@
+          })}
+          id={`conditional-${id}`}
+        >
+          {item.childrenWhenChecked}
+        </div>
+      )}
+    </React.Fragment>
+  );
+};
+
+const renderDivider = (
+  item: Divider,
+  index: number,
+  radioExtraAttributes: React.AriaAttributes & DataAttributes
+): JSX.Element => {
+  return (
+    <div
+      className="govuk-radios__divider"
+      key={index}
+      {...radioExtraAttributes}
+    >
+      {item.divider}
+    </div>
+  );
+};
+
+const renderItem = (
+  item: RadioButton | Divider,
+  name: string,
+  index: number,
+  idPrefix: string,
+  onChange?: (value: string) => void
+): JSX.Element => {
+  const radioExtraAttributes = Attributes.ariaAndData(item);
+  if ((item as Divider).divider) {
+    return renderDivider(item as Divider, index, radioExtraAttributes);
+  } else {
+    return renderRadio(
+      item as RadioButton,
+      name,
+      index,
+      idPrefix,
+      radioExtraAttributes,
+      onChange
+    );
+  }
+};
+
+export const Radios = (props: RadiosProps): JSX.Element => {
+  const {
+    className,
+    name,
+    items,
+    fieldset,
+    hint: radiosHint,
+    errorMessage,
+    onChange
+  } = props;
+  const formGroup = props.formGroup || {};
+  const idPrefix = props.idPrefix ? props.idPrefix : name;
+  const describedBy = [];
+  if (fieldset && fieldset.describedBy) {
+    describedBy.push(fieldset.describedBy);
+  }
+  let hintComponent: React.ReactNode;
+  let errorMessageComponent: React.ReactNode;
+  const hasConditional = items.some(
+    item => (item as RadioButton).childrenWhenChecked
+  );
+
+  if (radiosHint) {
+    const hintId = radiosHint.id ? radiosHint.id : `${idPrefix}-hint`;
+    describedBy.push(hintId);
+    hintComponent = <Hint id={hintId} {...radiosHint} />;
+  }
+
+  if (errorMessage) {
+    const errorId = errorMessage.id ? errorMessage.id : `${idPrefix}-error`;
+    describedBy.push(errorId);
+    errorMessageComponent = <ErrorMessage id={errorId} {...errorMessage} />;
+    formGroup.error = true;
+  }
+
+  const extraAttributes = Attributes.ariaAndData(props);
+  const radioClassNames = classNames("govuk-radios lbh-radios", className, {
+    "govuk-radios--conditional": hasConditional
+  });
+
+  const innerContent = (
+    <>
+      {radiosHint && hintComponent}
+      {errorMessage && errorMessageComponent}
+      <div
+        className={radioClassNames}
+        data-module={hasConditional ? "govuk-radios" : undefined}
+        {...extraAttributes}
+      >
+        {items.map((item, index) =>
+          renderItem(item, name, index, idPrefix, onChange)
+        )}
+      </div>
+    </>
+  );
+
+  return (
+    <FormGroup {...formGroup}>
+      {fieldset ? (
+        <Fieldset
+          describedBy={
+            describedBy.length > 0 ? describedBy.join(" ") : undefined
+          }
+          {...fieldset}
+        >
+          {innerContent}
+        </Fieldset>
+      ) : (
+        <>{innerContent}</>
```

Comment:
```suggestion
        innerContent
```

The extra fragment is unnecessary here. Not a requirement for merging, though.